bin/percona-qan-agent-installer/main.go

```diff
@@ -104,12 +105,22 @@ func main() {
 		os.Exit(1)
 	}
 
-	if !portSuffix.Match([]byte(args[0])) {
-		args[0] += ":" + DEFAULT_DATASTORE_PORT
+	qanAPIURL, err := url.Parse(args[0])
+	if err != nil {
+		log.Fatal("expected arg in the form [schema://]host[:port][path]")
+	}
+
+	if !portSuffix.MatchString(qanAPIURL.Host) {
+		qanAPIURL.Host += ":" + DEFAULT_DATASTORE_PORT
+	}
+
+	if !strings.HasPrefix(qanAPIURL.Scheme, "http") {
+		qanAPIURL.Scheme = "http://" + qanAPIURL.Scheme
 	}
 
 	agentConfig := &pc.Agent{
-		ApiHostname: args[0],
+		ApiHostname: qanAPIURL.Host,
+		ApiPath:     qanAPIURL.Path,
```

Comment:
Does this require a change to https://github.com/percona/pmm/blob/master/proto/config/config.go#L22 ? I mean adding `ApiPath` to type Agent?